src/output/tree.rs

```diff
@@ -298,6 +313,13 @@ impl<'module> MonoidalPrinter<'module> {
             Value::Seq(vals) => self.compile_seq(scope, vals, None),
             Value::Record(fields) => self.compile_record(scope, fields, None),
             Value::Variant(label, value) => self.compile_variant(scope, label, value, None),
+            Value::Mapped(orig, value) => {
+                if self.flags.collapse_mapped_values {
+                    self.compile_value(scope, value)
+                } else {
+                    self.compile_value(scope, orig)
```

Comment:
Is it proper to show only the original, and not the mapped value, if we have the `collapse_mapped_values` flag set? Or does this happen elsewhere?

Reply:
No I think we should show both the original and mapped values and currently it only shows the original, this is a limitation. I'm not sure what's the best way to show the original value, like how we should present both values, perhaps it could be on two lines? With names "orig" and "mapped" or something? It's a bit clunky...

Reply:
We could have something like

Inline Version

```
<fname> <- map (lambda) := <orig> ↦ <value>
```

Multiline Version

```
<fname> <- map (lambda) := <orig>
     ↳<value>
```

Reply:
Yes, although although most common is for orig to be multiline and value to be inline, for example base.u16 taking a tuple of two bytes to one U16.